src/main.tsx

```diff
@@ -1,14 +1,43 @@
+import {
+  ColorSchemeProvider,
+  createEmotionCache,
+  MantineProvider,
+} from '@mantine/core';
 import React from 'react';
 import ReactDOM from 'react-dom/client';
+import rtlPlugin from 'stylis-plugin-rtl';
 
 import { App } from './App';
 
+const rtlCache = createEmotionCache({
+  key: 'mantine-rtl',
+  stylisPlugins: [rtlPlugin],
+});
+
 const root = ReactDOM.createRoot(
   document.getElementById('root') as HTMLElement,
 );
 
 root.render(
   <React.StrictMode>
-    <App />
+    <ColorSchemeProvider
+      colorScheme="light"
+      toggleColorScheme={() => {
+        console.log('');
+      }}
```

Comment:
can't we omit this field?
is it required?

Reply:
then I'd say we remove the log and just make an empty function (maybe put a `NOTE` comment on it saying its required)
and if eslint has problem with empty function just ignore it with eslint ignore comment

Reply:
in this case, no